pkg/kube/kube_handler.go

```diff
@@ -567,7 +568,7 @@ func (a *userHandler) DeleteAppRepository(repoName, repoNamespace string) error
 	return err
 }
 
-func getValidationCli(appRepoBody io.ReadCloser, requestNamespace, kubeappsNamespace string) (*v1alpha1.AppRepository, HTTPClient, error) {
+func (a *userHandler) getValidationCli(appRepoBody io.ReadCloser, requestNamespace, kubeappsNamespace string) (*v1alpha1.AppRepository, HTTPClient, error) {
```

Comment:
Out of curiosity: why this change?
Edit: ah, for the test!

Reply:
this is executed when running the apprepo validation. We need now to fetch the secret from the namespace so we need to use the user clientset (so we can retrieve the credentials).